en/mkdocs.yml

```diff
@@ -48,14 +48,148 @@ theme:
 nav:
   - Home: index.md
   - Get Started:
-      - Introduction: get-started/overview.md
-      - Architecture:
-        - Architecture: get-started/architecture.md
-        - Provisioning Architecture: get-started/provisioning-architecture.md
-        - User Management Architecture: get-started/user-management-architecture.md
-      - About this Release: get-started/about-this-release.md
-      - Contribute: get-started/community.md
-      - Feature Tour: get-started/quick-start-guide.md
+      - Introduction: get-started/quick-start-guide.md
```

Comment:
How about this?
- Introduction
- Quick setup
- Try a use  case
   - Scenario - Overview
   - SSO
   - MFA
   - Identity Federation
   - Self sign-up